src/tests/mocks/bors.rs

```diff
@@ -32,6 +34,9 @@ use crate::{
 
 use super::pull_request::{GitHubPullRequestEventPayload, PullRequestChangeEvent};
 
+/// Counter used to generate incrementing SHA values for PR pushes.
+static PR_PUSH_SHA_COUNTER: AtomicU64 = AtomicU64::new(0);
```

Comment:
We should store this in `BorsTester`, not as a global variable. Tests run in parallel, so this could race and make tests flaky.